docker/spotifyd.Dockerfile

```diff
@@ -0,0 +1,40 @@
+#An extra layer to get around this bug https://github.com/docker/buildx/issues/395
+#It's there simply to download add required libraries for cargo build
+FROM --platform=$BUILDPLATFORM rust:bullseye AS rust_fix
+
+ENV USER=root
+ENV V_spotifyd=v0.3.5
+
+WORKDIR /usr/src/spotifyd
+RUN apt-get -y update && \
+    apt-get install --no-install-recommends -y apt-transport-https ca-certificates git && \
+    git clone --depth 1 --branch=${V_spotifyd} https://github.com/Spotifyd/spotifyd.git .
+
+# Don't do `cargo init` or --> error: `cargo init` cannot be run on existing Cargo packages
+# RUN cargo init
+RUN mkdir -p .cargo \
+  && cargo vendor > .cargo/config
+
+FROM rust:bullseye as build
+
+RUN apt-get -y update && \
+    apt-get install --no-install-recommends -y libasound2-dev build-essential pulseaudio libpulse-dev libdbus-1-dev
```

Comment:
Do we need `dbus`? The official website says, it's "experimental" and it only provides features we might not even need? Or do we actually rely on them?